helm/charts/determined/values.yaml

```diff
@@ -120,3 +120,6 @@ enterpriseEdition: false
 ## Configure whether we collect anonymous information about the usage of Determined.
 telemetry:
   enabled: true
+
+## A user friendly name to identify this cluster by.
+# clusterName: SF
```

Comment:
What is `SF`? I like `Dev` or `Prod`, because it's instantly easy to understand why/how you would want to use this field.

Reply:
+1 on updating this to `dev` or `prod`